app/src/lib/stores/commit-status-store.ts

```diff
@@ -187,6 +190,17 @@ export class CommitStatusStore {
   private backgroundRefreshHandle: number | null = null
   private refreshQueued = false
 
+  /**
+   * The repositories currently checked out ref and branch name. This is used to
+   * determine if we retrieve the action workflows to see if they exist for the
+   * checkruns, which we currently only do if the we are retrieve a status for a
+   * ref that is currently checked out - not for all the pull requests.
+   */
+  private checkedOutRef: {
+    ref: string
+    branchName: string
+  } | null = null
```

Comment:
Style nit, we can keep this compact

```suggestion
    private checkedOutRef: { ref: string; branchName: string } | null = null
```